build.sbt

```diff
@@ -11,6 +11,23 @@ ThisBuild / scalaVersion := scala213
 ThisBuild / crossScalaVersions := Seq(scala212, scala213)
 ThisBuild / tlCiReleaseBranches := Seq("master")
 ThisBuild / tlFatalWarningsInCi := false
+ThisBuild / tlMimaPreviousVersions ++= Set(
+  // manually added because tags are not v-prefixed
+  "0.1.0",
+  "0.1.1",
+  "0.2.0",
+  "0.2.1",
+  "0.3.0",
+  "0.4.0",
+  "0.4.1",
+  "0.5.0",
+  "0.6.0",
+  "0.6.1",
+  "0.6.1-14-g78cb9c2",
+  "0.7.0",
+  "0.7.0-M1",
+  "0.8.0"
+)
```

Comment:
I agree with @armanbilge, since this is still 'early-semvar' maintaining compat with the last minor version is sufficient.

Reply:
CI says it's fine. :-) I've also set `tlVersionIntroduced` which should trim that `Set` appropriately, right? I guess because of `tlVersionIntroduced` we could remove versions less than `0.3.0` here. The current `master` also checks bincompat since `0.3.0`.

Reply:
> I've also set `tlVersionIntroduced` which should trim that `Set` appropriately, right?

I can't remember tbh. I think that trims the information derived from tags, but anything added to `tlMimaPreviousVersions` has final say.

The definitive thing is to check `mimaPreviousArtifacts` and see what's in there.

Anyway, so long as CI is happy there's no problem with guaranteeing _extra_ compatibility beyond semver 😁  Cats does this to guarantee bincompat with 1.x for various modules, even in the 2.x series.